### DISCUSSION TITLE
`WARN DataSource: All paths were ignored`

### DISCUSSION THREAD

**Discussion by zmbc** [moj-analytical-services/splink#1821] [Q&A]
When running linkage with Spark, I get the message `WARN DataSource: All paths were ignored` many times during the linkage process. The warning always lists the location in the Spark checkpoint directory where a Splink dataframe (e.g. `__splink__df_predict_*`) was checkpointed.

I'm not too familiar with checkpointing but [this thread](https://community.databricks.com/t5/data-engineering/how-do-i-read-the-contents-of-a-hidden-file-in-a-spark-job/m-p/28035#M19873) suggests that Spark won't read "hidden files" which it defines as starting with a period or underscore.

My concern is that checkpointing isn't working for Splink tables at all, since these table names start with `__splink__` and therefore generate checkpoint file paths that are considered "hidden." I don't know how exactly to test whether or not this is the case. If this is just a red herring and everything is working as expected, a doc page or at least an answer here might help future travelers. 😃

**Comment by RobinL** [comment]
Just answering quickly from my phone:  yeah, pretty sure those warnings are false positives.  You're right we should figure out how to turn them off/document it
